unit/src/test/java/org/apache/falcon/unit/TestFalconUnit.java

```diff
@@ -493,6 +493,7 @@ public void testExtensionJobOperations() throws Exception {
         }
         enableExtension(TEST_EXTENSION);
 
+        Thread.sleep(10000);
```

Comment:
You could use the waitFor method instead, which uses polling rather than a fixed sleep time.